util/build_docs.py

```diff
@@ -211,24 +212,27 @@ def generate_tool_versions():
     exec(open(requirements_file).read(), globals())
 
     # And then write a version file for every tool.
-    for tool in __TOOL_REQUIREMENTS__:
+    for tool in __TOOL_REQUIREMENTS__:  # noqa: F821
         version_path = config["outdir-generated"].joinpath('version_' + tool + '.txt')
         version_path.parent.mkdir(parents=True, exist_ok=True)
         with open(str(version_path), mode='w') as fout:
-            fout.write(__TOOL_REQUIREMENTS__[tool])
+            fout.write(__TOOL_REQUIREMENTS__[tool])  # noqa: F821
 
 
-def is_hugo_extended():
-    args = ["hugo", "version"]
+def hugo_match_version(hugo_bin_path, version):
+    logging.info("Hugo binary path: %s", hugo_bin_path)
+    args = [str(hugo_bin_path), "version"]
     process = subprocess.run(args,
                              universal_newlines=True,
                              stdout=subprocess.PIPE,
                              check=True,
                              cwd=str(SRCTREE_TOP))
 
+    logging.info("Checking for correct Hugo version: %s", version)
     # Hugo version string example:
-    # Hugo Static Site Generator v0.59.0-1DD0C69C/extended linux/amd64 BuildDate: 2019-10-21T09:45:38Z
-    return bool(re.search("v\d+\.\d+\.\d+.*/extended", process.stdout))
+    # "Hugo Static Site Generator v0.59.0-1DD0C69C/extended linux/amd64 "
+    # "BuildDate: 2019-10-21T09:45:38Z"
```

Comment:
Don't wrap this line unless it's also wrapped in the original output.

Reply:
Ok. Appending `# noqa: E501` is fine? Will be an even longer line then

Reply:
Well, no good solutions here; but since we're regex-matching this line, it's good to have it in there as it actually shows up in the program output.

Reply:
Makes sense. Just noticed that I pushed with the quotes still in place. On the one hand this separates it from the comment, on the other it is again not what is returned, but I think it's fine.